test/suites/pvc/suite_test.go

```diff
@@ -0,0 +1,143 @@
+package integration
```

Comment:
Thoughts on broadening this suite to all storage integrations?
```suggestion
package storage
```